gamemode/core/libs/sh_notice.lua

```diff
@@ -26,7 +26,14 @@ if (SERVER) then
 else
 	-- List of notice panels.
 	ix.notices = ix.notices or {}
-
+	
+	-- Move all notices to their proper positions.
+	local function OrganizeNotices()
+		for k, v in ipairs(ix.notices) do
```

Comment:
it would benefit the performance to localize the scrW here, inside the function but outside of the loop